### PR TITLE
QOL fixes and closing issues

### LINKED ISSUE #154: Add LOFT role to roles.sql file

### PR DESCRIPTION
**Pull request by andrewjbe**
Closes #154

### CODE REVIEW

**Comment by andrewjbe** [comment]
Closes #156

**Comment by andrewjbe** [comment]
Closes #145

**Comment by andrewjbe** [comment]
Closes #91 and improves cli

**Comment by andrewjbe** [comment]
Closes #151

**Comment by andrewjbe** [comment]
Closes #122

**Comment by andrewjbe** [comment]
I want to close #128 and  #127 as well, but I need to discuss with @brancengregory first in order to understand the problem / desired behavior.

**Comment by andrewjbe** [comment]
Closes #127 and #128, also relevant to #158 but only for the `ojo_auth()` function.

**Comment by andrewjbe** [comment]
Confirmed that #158 is only relevant to `ojo_auth()`, so marking that closed too

File: R/ojo_auth.R [hunk 1]
```diff
@@ -29,21 +29,44 @@
...
-  clientcert <- file.path(home, ".postgresql/ojodb/client-cert.pem")
-  clientkey <- file.path(home, ".postgresql/ojodb/client-key.pk8")
+  renv <- fs::path(home, ".Renviron")
+  rootcert <- fs::path(home, ".postgresql/ojodb/server-ca.pem")
+  clientcert <- fs::path(home, ".postgresql/ojodb/client-cert.pem")
+  clientkey <- fs::path(home, ".postgresql/ojodb/client-key.pem")
+
+  # Check if SSL certs are in correct location; if not, throw error
+  if(!fs::file_exists(rootcert) |
+     !fs::file_exists(clientcert) |
```
Comment (brancengregory) [inline R/ojo_auth.R:39]: Changed from `{cli}` danger alert to `rlang::abort` (error out of the function)

**Review by brancengregory — APPROVED** [review]
Made small changes, only non-aesthetic one was making ojo_auth error out if the certs aren't found rather than just the danger message

**Pull request merged**